webview/platforms/cocoa.py

```diff
@@ -492,11 +492,10 @@ def restore(self):
         self.window.deminiaturize_(self)
 
     def move(self, x, y):
-        screen_frame = AppKit.NSScreen.mainScreen().frame()
```

Comment:
How about using `self.window.screen().frame()` here instead? This guarantees getting the correct window's frame instead of the frame of the active window, which might not be the one being moved.